docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -1 +1,162 @@
+? Initial project version 1.0.2
+Project initialized. Write a new contract in the contracts folder and run 'openzeppelin create' to deploy it.
+----
+
+Enable the Solidity Hot Loader in `client/config/webpack.js` by setting `disabled: false` in your favorite editor. We can disable this when we need.
+
+[source,js]
+----
+const solidityLoaderOptions = {
+  network: 'development',
+  // you can stop loader from automatic compile/push/upgrade
+  // action by setting disabled flag to true, but it will still
+  // serve .json files from file system
+  disabled: false,
+};
+
+module.exports = {
+  solidityLoader: {
+    test: /\.sol$/,
+    use: [
+      { loader: 'json-loader' },
+      {
+        loader: '@openzeppelin/solidity-loader',
+        options: solidityLoaderOptions,
+      },
+    ],
+  },
+  solidityLoaderOptions,
+};
+----
+
+== Counter contract
+Compile the Counter contract with OpenZeppelin CLI.
+
+[source,console]
+----
+$ npx oz compile
+✓ Compiled contracts with solc 0.5.13 (commit.5b0b510c)
+----
+
+Deploy the Counter contract to our local blockchain.
+
+Run `npx oz create` and use the interactive commands to deploy the Counter contract to the development network and initialize to a value e.g. `23`.
+
+[source,console]
+----
+$ npx oz create
+Nothing to compile, all contracts are up to date.
+? Pick a contract to instantiate Counter
+? Pick a network development
+✓ Added contract Counter
+✓ Contract Counter deployed
+All contracts have been deployed
+? Call a function to initialize the instance after creating it? Yes
+? Select which function * initialize(num: uint256)
+? num (uint256): 23
+✓ Setting everything up to create contract instances
+✓ Instance created at 0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+----
+
+== App
+We will now start the app.
+
+[source,console]
+----
+$ cd client
+$ npm run start
+----
+
+The app will open in a browser on http://localhost:3000/
+
+Click on Counter to go to http://localhost:3000/counter
+
+In MetaMask change your network to Localhost:8545 to connect to `ganache-cli`.
+
+== Fund MetaMask account
+If you need to fund your MetaMask account (e.g. it has a balance of 0 test Ether).
+
+From your project directory in another terminal, call `npx oz transfer` to transfer 0.5 (test) ether to your account in MetaMask.
+
+[source,console]
+----
+$ npx oz transfer
+? Pick a network development
+? Choose the account to send transactions from (0) 0xF983502f33E2c9E62d0465557446B86C595C4919
+? Enter the receiver account 0x77737a65C296012C67F8c7f656d1Df81827c9541
+? Enter an amount to transfer 0.5 ether
+✓ Funds sent. Transaction hash: 0x69a0ecc17b325a0e4cd414ca9d197a59e88a466c8d3b4d7a0a518ca384526af4
+----
+
+== Use the app
+In the app, press *Increase Counter by 1* button. Confirm the transaction in MetaMask and once the transaction is confirmed the app will have the updated counter value.
```

Comment:
I'd rephrase this and the following section to explain a bit more what's going on, instead of just giving instructions. For example after running the app we could go 'and what if we now wanted to change the contract? we'd have to do x and y and z, rerun our app, etc', so that the added value of the loader is better conveyed.

Reply:
Updated